AyuntamientoMadrid/consul
# Consul pull ⬇️

Where
=====
Pull changes from consul, including PR's:

**Features**
- Finish Image & Document attachment to Proposal & Investments https://github.com/consul/consul/pull/1915 && https://github.com/consul/consul/pull/1926
- Add Maps to Budget Investments and Proposals https://github.com/consul/consul/pull/1864
- Add tasks to Poll Shifts https://github.com/consul/consul/pull/1928

**Fixes**
- Fix Admin filters https://github.com/consul/consul/pull/1930

What
====
- Getting latest changes from Consul

How
===
- Carefully merging conflicts

Screenshots
===========
- Check PR's for more

Test
====
- Hopefully passing as well

Deployment
==========
- Activate or deactivate the each feature through its setting

For maps:
- Add secrets (check out /shared/config/secrets.yml in Pre)
- Mark the point in the map for Madrid as default location from /admin/settings

Warnings
========
- We had for a while the feature flag `Setting['feature.document_on_show']` to be able to hide the Upload Document buttons on Proposals and Investments, but with this PR those "bypasses" are not needed anymore. That's the reason for the extra-removal on the merge commit
